olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -32,7 +32,8 @@ metadata:
       of serverless applications and functions.
     repository: https://github.com/openshift-knative/serverless-operator
     support: Red Hat, Inc.
-  name: serverless-operator.v1.8.0-rc1
+    olm.skipRange: '>=1.7.2 <1.8.0'
```

Comment:
This `>=` probably means that even users being on 1.7.1 might be upgraded to 1.8.0? Maybe only for "manual" upgrades. I'm not sure about that.

Reply:
how so? I'd read this as "any user between (and including) 1.7.2 and 1.8.0" wheres before it would've been "between 1.7.3 and 1.8.0" I believe, which is not what we want? :thinking:

Reply:
Oh. I think you're right. I agree